### ISSUE TITLE
Dashboard Status Messages

### ISSUE THREAD

**Issue by lgebhardt** [brave-intl/publishers#214]
The dashboard area where "Verified" is shown (see below) is in need of functional clarification.
![screen shot 2017-10-25 at 8 11 14 am](https://user-images.githubusercontent.com/29154/31998012-25889e96-b95c-11e7-88b4-aff29c486a26.png)

Until recently (#178) the Verified message and green check were displayed only when the publisher was verified (a requirement to see the dashboard) and connected to uphold. When not connected to Uphold a message saying "Wallet not found, please Connect with Uphold to create one" was shown:

![screen shot 2017-10-24 at 11 27 14 am](https://user-images.githubusercontent.com/29154/31998229-0548a8fa-b95d-11e7-9a86-1674af2fdf9c.png)

On Slack @jenn-rhim said "i don’t think we should change what state we’d report under the domain name.  at least that’s not what i designated that area for." and "that’s strictly allocated to say something about the domain verification status.  whether that’s useful in the long run, we’d have to think about it and discuss ...but that is not meant to be a global notification area"

As of now some messages related to the Uphold connection process will still be shown in that area. We should define the desired behavior for these messages and possibly move them to the appropriate sub panel or other notification area.

**Comment by mrose17** [comment]
I'd like to see a tiny little uphold icon to the right of the green checkbox when connected and a lie statement symbol if statements are available for download.

**Comment by jenn-rhim** [comment]
What problems are we trying to solve?  I'm not convinced that these additional requests are solving any problems. Current dashboard is structured in a simple way that the user can find out what they need to know and that meets the requirements. I'd like to avoid organic and preemptive development where the value-add is not clear-- it'll be better handled when we hear user's feedback.

Here is the current implementation:

![publisher dashboard-preauth](https://user-images.githubusercontent.com/4369856/32004736-552aa112-b957-11e7-8ed4-7e3765669b4b.png)

![publisher dashboard-reauth](https://user-images.githubusercontent.com/4369856/32004753-616eb71a-b957-11e7-984e-07367566b706.png)

**Comment by mrose17** [comment]
i don't know whether anything is really needed at this stage; @lgebhardt - what do you think?

**Comment by lgebhardt** [comment]
Currently the message "Your Uphold wallet is being connected to your account" will be displayed in the case the publisher was sent to uphold, but has not completed the connection process. I'm not sure this message is needed. Should simply we remove the messaging from this area for now?

**Comment by jenn-rhim** [comment]
@lgebhardt can you take a screenshot of that or describe to me how that message box looks like and where exactly it comes up?

I do believe that we should indicate that the system is executing the user's request but the current message seems confusing. I'd rephrase it to "Sending you to Uphold for creating wallet..."
